src/index.js

```diff
@@ -32,16 +32,26 @@ _adapters._date.override({
 		return FORMATS;
 	},
 
-	parse: function(value/* , format */) {
+	parse: function(value, fmt) {
 		if (helpers.isNullOrUndef(value)) {
 			return null;
 		}
-
-		return parse(value);
+		const options = this.options ? this.options : {};
```

Comment:
`const` is not used anywhere in Chart.js (probably because its easier to disallow ES6 completely for IE11 suppport)
I thinks its fine here though.

Minor optimization: cache `typeof value` in a variable.

Reply:
yep, will change to `var`